### PR TITLE
Implemented Binary and Display traits for FixedBitSet.

### PR DESCRIPTION
**Pull request by Dolphindalt**
Providing some implementation for #34. Need feedback.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -356,6 +360,30 @@ impl FixedBitSet
...
+impl Binary for FixedBitSet {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), Error> {
+        let mut display_data = if f.alternate() {
+            String::from("0b")
+        } else {
+            String::new()
+        };
+
+        for block in self.data.iter().rev() {
+            display_data = display_data.add(&format!("{:b}", block));
```
Comment (bluss) [inline src/lib.rs:373]: nice. I'd suggest that we just write this piece by piece instead of creating an intermediate String.

File: src/lib.rs [hunk 2]
```diff
@@ -356,6 +360,32 @@ impl FixedBitSet
     }
 }
 
+#[cfg(feature = "std")]
```
Comment (bluss) [inline src/lib.rs:363]: These don't depend on the std feature and will work fine with core, in fact. That's just a smaller thing.

**Comment by bluss** [comment]
Looks good

File: src/lib.rs (hunk 2 above)
Comment (Dolphindalt) [reply src/lib.rs:363]: I should remove it though if it is true. On it.

**Comment by bluss** [comment]
Cool, thanks!

Let's be open to tweaks to the format in the future. Maybe `_` separators every 16 or 32 bits would be good to keep it readable.

**Pull request merged**